src/client/containers/CartPage/CartPage.Container.js

```diff
@@ -1,65 +1,91 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 import React from 'react';
 import './CartPage.Style.css';
-import { useParams } from 'react-router-dom';
 import Loader from '../../components/Loader/Loader.component';
 import ShoppingItem from '../../components/ShoppingItem/ShoppingItem';
-import DeliveryInfo from '../../components/DeliveryInfo/DeliveryInfo.component';
+// import DeliveryInfo from '../../components/DeliveryInfo/DeliveryInfo.component';
 import TotalPrice from '../../components/TotalPriceCard/TotalPriceCard.component';
 import ContactForm from '../../components/ContactForm/ContactForm.component';
 import ButtonComponent from '../../components/Button/Button.component';
-import { useFetchApi } from '../../hooks/UseFetchApi';
+// import { useFetchApi } from '../../hooks/UseFetchApi';
 import { useFirebase } from '../../firebase/FirebaseContext';
 import { PropTypes } from 'prop-types';
+import { useShoppingCartContext } from '../../context/shoppingCart/shoppingCartContext';
+import { useHistory } from 'react-router-dom';
 
 const CartPageContainer = ({ isAuthenticated }) => {
   const [cartItem, setCartItem] = React.useState([]);
-  const [user, setUser] = React.useState({});
-  const [userId, setUserId] = React.useState('');
+  const [IsLoading, setIsLoading] = React.useState(true);
+  // const [user, setUser] = React.useState({});
   const [total, setTotal] = React.useState(0);
-  const [itemCost, setItemCost] = React.useState([]);
-  const { id } = useParams();
+  // const [itemCost, setItemCost] = React.useState([]);
   const { auth } = useFirebase();
 
-  const orderData = useFetchApi(`orders/${id}`);
+  const { shoppingCart, changeProductQuantity } = useShoppingCartContext();
+  const history = useHistory();
+
+  /*
+  const userInfo = useFetchApi(`users/${userId}`);
+  
+  React.useEffect(() => {
+    if (!userInfo.isLoading) {
+      setUser(userInfo.data[0]);
+    }
+  }, [userInfo]);
+ */
```

Comment:
Can you delete these comment lines?

Reply:
you are right, I did not deleted because It will needed when I include DeliveyInfo Component. It will done in next PR.

Reply:
@varsha84 a PR is code to enter staging and comments don't belong on staging. If it was a draft PR, very well, it would be considered work in progress but a PR needs to be clean of comments.

Reply:
remove the commented code